File: comms/bulk-messaging/sender-pools/fetch-sender-pool-operation/fetch-sender-pool-operation.ts
import { TwilioClient } from "twilio-comms";

async function main() {
    const client = new TwilioClient({
        accountId: "TWILIO_ACCOUNT_SID",
        authToken: "TWILIO_AUTH_TOKEN",
    });
    await client.senderPools.fetchOperation("comms_operation_01h9krwprkeee8fzqspvwy6nq8");
}
main();

File: comms/bulk-messaging/sender-pools/fetch-sender-pool-operation/fetch-sender-pool-operation.cs
using Twilio.Comms;
using System.Threading.Tasks;

public partial class Examples
{
    public async Task Example() {
        var client = new TwilioComms(
            accountId: "TWILIO_ACCOUNT_SID",
            authToken: "TWILIO_AUTH_TOKEN"
        );

        await client.SenderPools.FetchOperationAsync(
            "comms_operation_01h9krwprkeee8fzqspvwy6nq8"
        );
    }

}
